tests/src/Discovery/PathFinderBaseTest.php

```diff
@@ -17,6 +17,32 @@
 class PathFinderBaseTest extends \PHPUnit_Framework_TestCase {
 
   /**
+   * Tests that constructor works properly.
+   *
+   * @covers ::__construct()
+   */
+
+  public function testConstructor() {
+    $pathFinder = new FakePathFinder(['./testFolder/']);
+    $property = new \ReflectionProperty($pathFinder, 'path');
+    $property->setAccessible(true);
+    $value = $property->getValue($pathFinder);
+    $this->assertEquals('./testFolder/', $value);
+  }
+
+  /**
+   * Tests that PathFinderBase::requireFile() works properly.
+   *
+   * @covers ::requireFile()
+   */
+  public function testRequireFile() {
+    $pathFinder = new FakePathFinder(['']);
+    $pathFinder->requireFile(realpath('data/acme.inc'));
```

Comment:
You need to provide a seed in here. A seed is a path to a file that **we know** is inside of a Drupal installation.

In this case it doesn't matter the value here, since we are not faking a drupal tree stucture … yet. We will need to for the `PathFinderContrib` and `PathFinderCore`.